db/migrate/20180228222337_add_periods_location_to_sections.rb

```diff
@@ -0,0 +1,5 @@
+class AddPeriodsLocationToSections < ActiveRecord::Migration[5.1]
+  def change
+    add_column :sections, :periods_location, :string, array: true, default: []
```

Comment:
can we add a `null: false` here? Then be sure to run the migration again.

Reply:
Sure thing. Would I just do this on my local machine, then re-push to my fork?

Reply:
yup!